### ISSUE TITLE
About performance on market1501 for global learning and mutual learning 

### ISSUE THREAD

**Issue by vincentman** [huanghoujing/AlignedReID-Re-Production-Pytorch#69]
I try to train with global learning and mutual learning on market1501, and then mAP for model 1 is 68.82%, mAP for model 2 is 68.56%. These performances are worse than yours(your mAP is around 75%).

Snapshot from AlignedReID-Scores.xlsx:
![image](https://user-images.githubusercontent.com/10985954/52685193-e0306500-2f83-11e9-9d79-6db2b63d6136.png)

What are your parameters for training? I post my training script as following:
python3 script/experiment/train_ml.py \
-d "((0,), (1,))" \
-r 1 \
--num_models 2 \
--dataset market1501 \
--ids_per_batch 32 \
--ims_per_id 4 \
--normalize_feature false \
-gm 0.3 \
-glw 1 \
-llw 0 \
-idlw 0 \
-pmlw 0 \
-gdmlw 1 \
-ldmlw 0 \
--base_lr 2e-4 \
--lr_decay_type exp \
--exp_decay_at_epoch 151 \
--total_epochs 300
